docs/patterns/errorpages.rst

```diff
@@ -97,3 +97,31 @@ An example template might be this:
       <p>What you were looking for is just not there.
       <p><a href="{{ url_for('index') }}">go somewhere nice</a>
     {% endblock %}
+
+
+Handling API Errors with Abort
+------------------------------
+
+When using Flask for web APIs, handling errors is as simple as the previous examples but with minor changes.
+
+Register the error handler::
+
+    from flask import jsonify
+
+    @app.errorhandler(404)
+    def resource_not_found(e):
+        # if passing in an Exception object directly, you may need to convert it to a string
+        stringified_exception = str(e)
+        return jsonify(error=stringified_exception)
```

Comment:
This no longer returns a 404 status.

Reply:
Thanks. I'll rebase to squash these commits down and update.